src/src/tx_main.cpp

```diff
@@ -343,22 +350,16 @@ void ICACHE_RAM_ATTR SendRCdataToRF()
   }
   else
   {
-    #if defined HYBRID_SWITCHES_8
```

Comment:
I think HYBRID_SWITCHES_8 and SEQ_SWITCHES need to be added back, but moved to the else statement with Generate4ChannelData_11bit();

Reply:
Something screwy has happened here with the merge from Master. I'll fix it up.